frontend/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -14,104 +17,122 @@ message-base(v-bind='$props' @wrapperAction='action')
-      passed: L('Left {title}', summary),
-      default: L('No members were removed.')
+    [PROPOSAL_REMOVE_MEMBER]: {
+      [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to remove members from the group.', options),
+      [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to remove members from the group.', options),
+      [PROPOSAL_VARIANTS.ACCEPTED]: L('Left {title}', options),
+      [PROPOSAL_VARIANTS.REJECTED]: L('No members were removed.'),
+      [PROPOSAL_VARIANTS.EXPIRED]: L('No members were removed.')
     },
-    'group-setting-change': {
+    [PROPOSAL_GROUP_SETTING_CHANGE]: {
       mincomeAmount: {
-        open: L('{from} wants to change the change the groups mincome. ', interaction),
-        passed: L('The groups mincome changed.'),
-        default: L('The group mincome hasn\'t changed.')
+        [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to change the groups mincome. ', options),
+        [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to change the groups mincome. ', options),
+        [PROPOSAL_VARIANTS.ACCEPTED]: L('The groups mincome changed.'),
+        [PROPOSAL_VARIANTS.REJECTED]: L('The group mincome hasn\'t changed.'),
+        [PROPOSAL_VARIANTS.EXPIRED]: L('The group mincome hasn\'t changed.')
       }
     },
-    'proposal-setting-change': {
+    [PROPOSAL_PROPOSAL_SETTING_CHANGE]: {
       votingRule: {
-        open: L('{from} wants to change the groups voting rules. ', interaction),
-        passed: L('The groups voting rules changed'),
-        default: L('The groups voting rules hasn\'t changed.')
+        [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to change the groups voting rules. ', options),
+        [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to change the groups voting rules. ', options),
+        [PROPOSAL_VARIANTS.ACCEPTED]: L('The groups voting rules changed'),
+        [PROPOSAL_VARIANTS.REJECTED]: L('The groups voting rules hasn\'t changed.'),
+        [PROPOSAL_VARIANTS.EXPIRED]: L('The groups voting rules hasn\'t changed.')
       },
       votingSystem: {
-        open: L('{from} wants to change the groups voting system. ', interaction),
-        passed: L('The groups voting system changed.'),
-        default: L('The groups voting system hasn\'t changed.')
+        [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to change the groups voting system. ', options),
+        [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to change the groups voting system. ', options),
+        [PROPOSAL_VARIANTS.ACCEPTED]: L('The groups voting system changed.'),
+        [PROPOSAL_VARIANTS.REJECTED]: L('The groups voting system hasn\'t changed.'),
+        [PROPOSAL_VARIANTS.EXPIRED]: L('The groups voting system hasn\'t changed.')
       }
+    },
+    [PROPOSAL_GENERIC]: {
+      [PROPOSAL_VARIANTS.CREATED]: L('{from} created a proposal. "{title}"', options),
+      [PROPOSAL_VARIANTS.EXPIRING]: L('{from} created a proposal. "{title}"', options),
```

Comment:
Great work on adding the title into this @Silver-IT! 🔥 

I just realized though, that the message I came up with isn't very good. It makes it sound like the proposal was just created instead of expiring.

So how about we change the message from:

```js
L('{from} created a proposal. "{title}"', options)
```

To:

```js
L('Proposal from {from} is expiring: "{title}"', options)
```

We should also make similar changes to the other expiring messages.